pontoon/sites/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.conf.urls import patterns, url
+import views
+
+urlpatterns = patterns('',
+    # Sites pages
+    url(r'^tiles/$', views.sites_snippet_page,
+        kwargs=dict(template="tiles.html",
+                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxtiles/en-US/",
+                    default_filename="tiles.lang"),
+        name='sites-tiles'),
+    url(r'^snippets/$', views.sites_snippet_page,
+        kwargs=dict(template="snippets.html",
+                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/snippets/en-US/",
+                    default_filename="jan2014.lang"),
+        name='sites-tiles'),
```

Comment:
Okay, i'll fix these bugs tomorrow. Thx for review.

On Mon, Aug 31, 2015 at 8:02 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/sites/urls.py
> <https://github.com/mozilla/pontoon/pull/137#discussion_r38339703>:
>
> > @@ -0,0 +1,21 @@
> > +from django.conf.urls import patterns, url
> > +import views
> > +
> > +urlpatterns = patterns('',
> > +    # Sites pages
> > +    url(r'^tiles/$', views.sites_snippet_page,
> > +        kwargs=dict(template="tiles.html",
> > +                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxtiles/en-US/",
> > +                    default_filename="tiles.lang"),
> > +        name='sites-tiles'),
> > +    url(r'^snippets/$', views.sites_snippet_page,
> > +        kwargs=dict(template="snippets.html",
> > +                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/snippets/en-US/",
> > +                    default_filename="jan2014.lang"),
> > +        name='sites-tiles'),
>
> sites-snippets
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/137/files#r38339703>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.